subsys/bluetooth/controller/include/ll.h

```diff
@@ -7,6 +7,12 @@
 
 #define LL_VERSION_NUMBER BT_HCI_VERSION_5_1
 
+enum ll_txp {
```

Comment:
As discussed this struct has been renamed as `struct bt_hci_vs_handle_type` [here](https://github.com/zephyrproject-rtos/zephyr/pull/19779/commits/b8c7a15362e46dbcc1238e7bf79694779119a062#diff-65c5dce42a06bd585801995fc7a03ad0R10) so that the pattern could be reused by possible future HCI VS commands that need to implement functionality through the lower layers.